suggest.rb

```diff
@@ -138,21 +138,40 @@ def search_string_in_files(search_string)
 puts "searching for key words in searchable files"
 
 search_results = []
-chosen_key_words.each { |key_word| search_results << search_string_in_files(key_word) }
-
-ap search_results
+chosen_key_words.each { |key_word| search_results << search_string_in_files(key_word, filename) }
```

Comment:
You can use `Array#map` here if you want
```suggestion
search_results = chosen_key_words.map { |key_word| search_string_in_files(key_word, filename) }
```

Reply:
Excellent, thanks!